client/src/main/java/gui/data/Player.java

```diff
@@ -0,0 +1,41 @@
+package gui.data;
+
+import java.util.Objects;
+
+public class Player {
```

Comment:
good candidate for AutoValue :)

Reply:
Thank you for suggestion. We don't use for now AutoValue, maybe in the next project. :)